src/main/resources/com/google/api/codegen/readme_rst.snip

```diff
@@ -9,6 +9,8 @@
   ==================================================================================================
 
   Idiomatic {@metadata.targetLanguage} client for `{@metadata.fullName}`_
+  -----------------------------------------------------------------------
```

Comment:
optional: line 37 of this file incorrectly refers to `monitoring` -- feel free to address in a follow-up PR if you prefer, since it's not directly linked to this one.